Makefile

```diff
@@ -318,6 +320,11 @@ PORTLAYER_DEPS ?= lib/apiservers/portlayer/swagger.json \
 				  lib/apiservers/portlayer/restapi/options/*.go \
 				  lib/apiservers/portlayer/restapi/handlers/*.go
 
+$(admiralapi-client): lib/config/dynamic/admiral/swagger.json $(SWAGGER)
```

Comment:
I just remembered that there are some errors in the swagger file for some xenon types. The bug is actually in the swagger-xenon adapter. I have fixed that upstream, and am in the process of merging into the xenon-utils repo, but that fix has to make it into Admiral as well -- that is going to be too late for this PR.